main.go

```diff
@@ -491,3 +500,28 @@ func extractChdirOption(args []string) (string, []string, error) {
 	copy(newArgs[argPos:], args[argPos+1:])
 	return argValue, newArgs, nil
 }
+
+// Creates the the configuration directory.
+// `configDir` should refer to `~/.terraform.d` or its equivalent
+// on non-UNIX platforms.
+func createConfigDir(configDir string) error {
+	configDirInfo, err := os.Stat(configDir)
+	if os.IsNotExist(err) {
```

Comment:
Isn't better to just try create the directory directly and handle the error? This if is not safe and the directory can be created (even though chances are low) after the IsNotExists and before the Mkdir

Reply:
@ilmax Makes sense. I've removed the os.Stat call.